### DISCUSSION TITLE
Enhancement: Option to auto pickup auto foraged plants from bushes.

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#5176] [General]
When you auto forage trees or Everything, it will place the products in your inventory. Bushes don't do that. That's because bushes give random items. Suggest to add a setting to autopickup this stuff as default behavior changing may not be desirable here. It's annoying to forage bushes and your products go on the floor, like wild vegetables and buckwheat.
![image](https://github.com/user-attachments/assets/66d0c864-2dfb-4d00-8073-74cabbfd44d3)

**Comment by scarf005** [comment]
this sounds like a good QoL feature, let's make it an issue

**Comment by scarf005** [comment]
will continue in #5177
